### ISSUE TITLE
Header UI elements not accessible for screen readers on Android 7.1 using Talkback 6.2.0

### ISSUE THREAD

**Issue by icelander** [mattermost/mattermost-server#9036]
*Summary*

The header UI elements for the RHB, Channel Name, Search, and application menus are not labeled correctly for a screen reader.

*Steps to Reproduce*

1. Use Version 1.9.2 of the Mattermost application on Android 7.1 and Talkback 6.2.0

*Expected Behavior*

Top menu items are labeled properly

*Observed Behavior*

They're not labeled properly and do not show up as buttons.

*Possible Fixes*

**Comment by amyblais** [comment]
@icelander Please let me know if you were able to repro this or if you need QA's help to test it out (I don't have Android to test it)?

**Comment by nktech1135** [comment]
Hi.
I'm the one who origionally reported this through the email link on the play store. is there anything i can do to help with this?

Thanks.

**Comment by amyblais** [comment]
Hi @nktech1135! I created a Jira ticket here for our engineers to take a look: https://mattermost.atlassian.net/browse/MM-11302.

We'll let you know if we need further information!

**Comment by amyblais** [comment]
Closing as this was opened as a Help Wanted issue here: https://github.com/mattermost/mattermost-server/issues/13983.
